applications/trilinos_application/external_includes/amgcl_mpi_solve_functions.h

```diff
@@ -74,7 +64,10 @@ AMGCLBlockSolve(
     int verbosity_level
     )
 {
-    amgclParams.put("precond.coarsening.aggr.block_size",1);
+    if(amgclParams.get<std::string>("precond.class") != "amg")
+        amgclParams.erase("precond.coarsening");
+    else
+        amgclParams.put("precond.coarsening.aggr.block_size",1);
```

Comment:
this needs to be always 1? asking just to be sure it is not a copy-paste error

Reply:
This is here for the same reasons it is in the serial case: https://github.com/KratosMultiphysics/Kratos/blob/master/kratos/linear_solvers/amgcl_solver.h#L137-L140

When we are using block values to solve the system, we generally do not need to set `block_size`.

Reply:
ok, sorry for the noise. approving